sn_client/src/api.rs

```diff
@@ -74,9 +89,16 @@ impl Client {
 
         info!("Startup a client with peers {peers:?} and local {local:?} flag");
         info!("Starting Kad swarm in client mode...");
+        debug!("Starting Kad swarm in client mode.1..");
+        trace!("Starting Kad swarm in client mode..2.");
```

Comment:
don't get it, why have 3 different client mode index with three different log level ?

Reply:
ah, that should be cleaned up!